### PR TITLE
add logs to fire base action and some minor variable name change

### PR DESCRIPTION
**Pull request by gauravsaralMs**
2 changes in this PR
- adding extra logs in fire base action so that in case of issue debugging can be easy and developers can see how action is working when actually deployed
- minor update in variable name to reflect changes being made in looker code base

### CODE REVIEW

File: lib/actions/firebase/firebase.js [hunk 1]
```diff
@@ -104,34 +111,39 @@ class FirebaseAction extends Hub.Action {
...
     }
-    sendMessageToDevice(deviceId, payload, options) {
+    sendMessageToDevice(deviceId, webhookId, payload, options) {
         return __awaiter(this, void 0, void 0, function* () {
             return new Promise((resolve, reject) => {
                 FirebaseAction.setFirebaseClient();
                 firebaseAdmin.messaging().sendToDevice(deviceId, payload, options)
-                    .then((_) => {
+                    .then((response) => {
+                    winston.info(`${LOG_PREFIX} notification sent to firebase. ${JSON.stringify(response)}`, { webhookId });
```
Comment (gauravsaralMs) [inline lib/actions/firebase/firebase.js:142]: @phillipperalez : AH++ has info level logging turned on ?

Comment (gauravsaralMs) [reply lib/actions/firebase/firebase.js:142]: or should I move this to warn

Comment (phillipperalez) [reply lib/actions/firebase/firebase.js:142]: info is valid and logged

Comment (gauravsaralMs) [reply lib/actions/firebase/firebase.js:142]: 👍🏼 
then this PR is ready for review and merge 😄

**Review by phillipperalez — APPROVED** [review]

**Pull request merged**